autoinstrumentation/nodejs/package.json

```diff
@@ -14,9 +14,14 @@
     "typescript": "^4.4.4"
   },
   "dependencies": {
-    "@opentelemetry/api": "1.1.0",
-    "@opentelemetry/auto-instrumentations-node": "0.31.0",
-    "@opentelemetry/exporter-trace-otlp-grpc": "0.31.0",
-    "@opentelemetry/sdk-node": "0.31.0"
+    "@opentelemetry/api": "1.2.0",
+    "@opentelemetry/auto-instrumentations-node": "0.33.1",
+    "@opentelemetry/exporter-trace-otlp-grpc": "0.33.0",
+    "@opentelemetry/resource-detector-alibaba-cloud": "0.27.2",
+    "@opentelemetry/resource-detector-aws": "1.1.2",
+    "@opentelemetry/resource-detector-container": "0.1.2",
```

Comment:
can you make use of the latest version (0.2.0), this contains an update for containers running on cgroupv2